nemo/core/classes/mixins/adapter_mixins.py

```diff
@@ -834,3 +834,17 @@ def update_adapter_cfg(self, cfg: DictConfig):
         for module in self.modules():  # access PT subclass method via inheritance
             if isinstance(module, AdapterModuleMixin):
                 module.adapter_cfg = cfg
+
+    @property
+    def adapter_module_names(self) -> List[str]:
+        """
+        List of valid adapter modules that are supported by the model.
+
+        **Note**: Subclasses should override this property and return a list of str names, of all the modules
+            that they support, which will enable users to determine where to place the adapter modules.
+
+        Returns:
+            A list of str, one for each of the adapter modules that are supported. By default, the subclass
+            should support the "global adapter" ('').
+        """
+        return ['']
```

Comment:
it is considered to be an empty list?

Reply:
It is considered the "default" and "global" adapter cases. Each domain can chose to support such "default" or "global" adapter cases. For example, ASR allows default adapters, which default to encoder adapters. They are also global in the sense that you don't provide a module name to them.